### ISSUE TITLE
MissingReferenceException: The object of type 'SpriteRenderer' has been destroyed but you are still trying to access it.

### ISSUE THREAD

**Issue by mofr** [mofr/Diablerie#86]
It happens right after selecting the character in the main menu.

**Comment by Agiofobia** [comment]
There must be a code reference to the SpriteRenderer such as a `GetComponent<SpriteRenderer>()`. Or you might be trying to access It with an animation

**Comment by mofr** [comment]
Yeah, something like this, but I didn't find exactly back in the time. Thank you

**Comment by Agiofobia** [comment]
I just realized this project is from 2019 :D
